entrypoint.sh.erb

```diff
@@ -1,20 +1,8 @@
 <% is_alpine = (dockerfile.split("/").last.split("-").first == "alpine") %>
-#!/usr/bin/dumb-init /bin/sh
-
-uid=${FLUENT_UID:-1000}
-
-# check if a old fluent user exists and delete it
-cat /etc/passwd | grep fluent
-if [ $? -eq 0 ]; then
-    deluser fluent
-fi
-
-# (re)add the fluent user with $FLUENT_UID
 <% if is_alpine %>
-adduser -D -g '' -u ${uid} -h /home/fluent fluent
+#!/bin/sh
 <% else %>
-useradd -u ${uid} -o -c "" -m fluent
-export HOME=/home/fluent
+#!/bin/bash
```

Comment:
If you wanted to use `#!/bin/sh` on both Alpine and Debian, I think all you'd need to adjust is changing `[ "${1:0:1}" = '-' ]` into something like `[ "${1#-}" != "$1" ]` instead.

Reply:
Will fix.